"make release" fails: ln: failed to create symbolic link './clang': File exists

Full log:
```
$~/sandbox/gopath/src/github.com/cilium/pwru [09:54:20] @ main
> make release
docker run \
        --rm \
        --workdir /pwru \
        --volume `pwd`:/pwru docker.io/library/golang:1.18.3 \
        sh -c "apt update && apt install -y make git clang-13 llvm && \
                ln -s  /usr/bin/clang && \
                git config --global --add safe.directory /pwru && \
                make libpcap.a && \
                make local-release VERSION=v0.0.9-38-gf7f9bdb"

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

Get:1 http://deb.debian.org/debian bullseye InRelease [116 kB]
Get:2 http://security.debian.org/debian-security bullseye-security InRelease [48.4 kB]
Get:3 http://deb.debian.org/debian bullseye-updates InRelease [44.1 kB]
Get:4 http://security.debian.org/debian-security bullseye-security/main amd64 Packages [252 kB]
Get:5 http://deb.debian.org/debian bullseye/main amd64 Packages [8183 kB]
Get:6 http://deb.debian.org/debian bullseye-updates/main amd64 Packages [14.8 kB]
Fetched 8658 kB in 4s (2095 kB/s)
Reading package lists...
Building dependency tree...
Reading state information...
53 packages can be upgraded. Run 'apt list --upgradable' to see them.

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

Reading package lists...
Building dependency tree...
Reading state information...
make is already the newest version (4.3-4.1).
The following additional packages will be installed:
  binfmt-support icu-devtools lib32gcc-s1 lib32stdc++6 libc-dev-bin libc6
  libc6-dev libc6-i386 libclang-common-13-dev libclang-cpp11 libclang-cpp13
  libclang1-13 libffi-dev libgc1 libicu-dev libicu67 libllvm11 libllvm13
  libncurses-dev libncurses6 libncursesw6 libobjc-10-dev libobjc4 libpfm4
  libpipeline1 libtinfo-dev libtinfo6 libxml2 libxml2-dev libyaml-0-2 libz3-4
  libz3-dev llvm-11 llvm-11-dev llvm-11-runtime llvm-11-tools llvm-13
  llvm-13-dev llvm-13-linker-tools llvm-13-runtime llvm-13-tools llvm-runtime
  python3-pkg-resources python3-pygments python3-yaml
Suggested packages:
  clang-13-doc gettext-base git-daemon-run | git-daemon-sysvinit git-doc
  git-el git-email git-gui gitk gitweb git-cvs git-mediawiki git-svn glibc-doc
  libc-l10n locales manpages-dev icu-doc ncurses-doc llvm-11-doc llvm-13-doc
  python3-setuptools python-pygments-doc ttf-bitstream-vera
Recommended packages:
  patch less manpages manpages-dev libc-devtools libnss-nis libnss-nisplus
  libgpm2
The following NEW packages will be installed:
  binfmt-support clang-13 icu-devtools lib32gcc-s1 lib32stdc++6 libc6-i386
  libclang-common-13-dev libclang-cpp11 libclang-cpp13 libclang1-13 libffi-dev
  libgc1 libicu-dev libicu67 libllvm11 libllvm13 libncurses-dev libobjc-10-dev
  libobjc4 libpfm4 libpipeline1 libtinfo-dev libxml2 libxml2-dev libyaml-0-2
  libz3-4 libz3-dev llvm llvm-11 llvm-11-dev llvm-11-runtime llvm-11-tools
  llvm-13 llvm-13-dev llvm-13-linker-tools llvm-13-runtime llvm-13-tools
  llvm-runtime python3-pkg-resources python3-pygments python3-yaml
The following packages will be upgraded:
  git libc-dev-bin libc6 libc6-dev libncurses6 libncursesw6 libtinfo6
7 upgraded, 41 newly installed, 0 to remove and 46 not upgraded.
Need to get 196 MB of archives.
After this operation, 1110 MB of additional disk space will be used.
Get:1 http://deb.debian.org/debian bullseye/main amd64 libc6-dev amd64 2.31-13+deb11u6 [2360 kB]
Get:2 http://deb.debian.org/debian bullseye/main amd64 libc-dev-bin amd64 2.31-13+deb11u6 [276 kB]
Get:3 http://deb.debian.org/debian bullseye/main amd64 libc6 amd64 2.31-13+deb11u6 [2827 kB]
Get:4 http://deb.debian.org/debian bullseye/main amd64 libncurses6 amd64 6.2+20201114-2+deb11u1 [102 kB]
Get:5 http://deb.debian.org/debian bullseye/main amd64 libncursesw6 amd64 6.2+20201114-2+deb11u1 [132 kB]
Get:6 http://deb.debian.org/debian bullseye/main amd64 libtinfo6 amd64 6.2+20201114-2+deb11u1 [341 kB]
Get:7 http://deb.debian.org/debian bullseye/main amd64 libpipeline1 amd64 1.5.3-1 [34.3 kB]
Get:8 http://deb.debian.org/debian bullseye/main amd64 binfmt-support amd64 2.2.1-1+deb11u1 [66.8 kB]
Get:9 http://deb.debian.org/debian bullseye/main amd64 libicu67 amd64 67.1-7 [8622 kB]
Get:10 http://deb.debian.org/debian bullseye/main amd64 libxml2 amd64 2.9.10+dfsg-6.7+deb11u4 [693 kB]
Get:11 http://deb.debian.org/debian bullseye/main amd64 libz3-4 amd64 4.8.10-1 [6949 kB]
Get:12 http://deb.debian.org/debian bullseye/main amd64 libllvm13 amd64 1:13.0.1-6~deb11u1 [20.1 MB]
Get:13 http://deb.debian.org/debian bullseye/main amd64 libclang-cpp13 amd64 1:13.0.1-6~deb11u1 [10.7 MB]
Get:14 http://deb.debian.org/debian bullseye/main amd64 libgc1 amd64 1:8.0.4-3 [239 kB]
Get:15 http://deb.debian.org/debian bullseye/main amd64 libobjc4 amd64 10.2.1-6 [41.8 kB]
Get:16 http://deb.debian.org/debian bullseye/main amd64 libobjc-10-dev amd64 10.2.1-6 [170 kB]
Get:17 http://deb.debian.org/debian bullseye/main amd64 libc6-i386 amd64 2.31-13+deb11u6 [2615 kB]
Get:18 http://deb.debian.org/debian bullseye/main amd64 lib32gcc-s1 amd64 10.2.1-6 [49.4 kB]
Get:19 http://deb.debian.org/debian bullseye/main amd64 lib32stdc++6 amd64 10.2.1-6 [510 kB]
Get:20 http://deb.debian.org/debian bullseye/main amd64 libclang-common-13-dev amd64 1:13.0.1-6~deb11u1 [5602 kB]
Get:21 http://deb.debian.org/debian bullseye/main amd64 llvm-13-linker-tools amd64 1:13.0.1-6~deb11u1 [1253 kB]
Get:22 http://deb.debian.org/debian bullseye/main amd64 libclang1-13 amd64 1:13.0.1-6~deb11u1 [6187 kB]
Get:23 http://deb.debian.org/debian bullseye/main amd64 clang-13 amd64 1:13.0.1-6~deb11u1 [123 kB]
Get:24 http://deb.debian.org/debian bullseye/main amd64 git amd64 1:2.30.2-1+deb11u2 [5518 kB]
Get:25 http://deb.debian.org/debian bullseye/main amd64 icu-devtools amd64 67.1-7 [201 kB]
Get:26 http://deb.debian.org/debian bullseye/main amd64 libllvm11 amd64 1:11.0.1-2 [17.9 MB]
Get:27 http://deb.debian.org/debian bullseye/main amd64 libclang-cpp11 amd64 1:11.0.1-2 [9632 kB]
Get:28 http://deb.debian.org/debian bullseye/main amd64 libffi-dev amd64 3.3-6 [56.5 kB]
Get:29 http://deb.debian.org/debian bullseye/main amd64 libicu-dev amd64 67.1-7 [9597 kB]
Get:30 http://deb.debian.org/debian bullseye/main amd64 libncurses-dev amd64 6.2+20201114-2+deb11u1 [344 kB]
Get:31 http://deb.debian.org/debian bullseye/main amd64 libpfm4 amd64 4.11.1+git32-gd0b85fb-1 [286 kB]
Get:32 http://deb.debian.org/debian bullseye/main amd64 libtinfo-dev amd64 6.2+20201114-2+deb11u1 [940 B]
Get:33 http://deb.debian.org/debian bullseye/main amd64 libxml2-dev amd64 2.9.10+dfsg-6.7+deb11u4 [790 kB]
Get:34 http://deb.debian.org/debian bullseye/main amd64 libyaml-0-2 amd64 0.2.2-1 [49.6 kB]
Get:35 http://deb.debian.org/debian bullseye/main amd64 libz3-dev amd64 4.8.10-1 [90.8 kB]
Get:36 http://deb.debian.org/debian bullseye/main amd64 llvm-11-runtime amd64 1:11.0.1-2 [212 kB]
Get:37 http://deb.debian.org/debian bullseye/main amd64 llvm-runtime amd64 1:11.0-51+nmu5 [8532 B]
Get:38 http://deb.debian.org/debian bullseye/main amd64 llvm-11 amd64 1:11.0.1-2 [8463 kB]
Get:39 http://deb.debian.org/debian bullseye/main amd64 llvm amd64 1:11.0-51+nmu5 [10.7 kB]
Get:40 http://deb.debian.org/debian bullseye/main amd64 python3-pkg-resources all 52.0.0-4 [190 kB]
Get:41 http://deb.debian.org/debian bullseye/main amd64 python3-pygments all 2.7.1+dfsg-2.1 [657 kB]
Get:42 http://deb.debian.org/debian bullseye/main amd64 python3-yaml amd64 5.3.1-5 [138 kB]
Get:43 http://deb.debian.org/debian bullseye/main amd64 llvm-11-tools amd64 1:11.0.1-2 [373 kB]
Get:44 http://deb.debian.org/debian bullseye/main amd64 llvm-11-dev amd64 1:11.0.1-2 [28.0 MB]
Get:45 http://deb.debian.org/debian bullseye/main amd64 llvm-13-runtime amd64 1:13.0.1-6~deb11u1 [265 kB]
Get:46 http://deb.debian.org/debian bullseye/main amd64 llvm-13 amd64 1:13.0.1-6~deb11u1 [11.0 MB]
Get:47 http://deb.debian.org/debian bullseye/main amd64 llvm-13-tools amd64 1:13.0.1-6~deb11u1 [421 kB]
Get:48 http://deb.debian.org/debian bullseye/main amd64 llvm-13-dev amd64 1:13.0.1-6~deb11u1 [31.9 MB]
debconf: delaying package configuration, since apt-utils is not installed
Fetched 196 MB in 2min 33s (1281 kB/s)
(Reading database ... 15718 files and directories currently installed.)
Preparing to unpack .../libc6-dev_2.31-13+deb11u6_amd64.deb ...
Unpacking libc6-dev:amd64 (2.31-13+deb11u6) over (2.31-13+deb11u3) ...
Preparing to unpack .../libc-dev-bin_2.31-13+deb11u6_amd64.deb ...
Unpacking libc-dev-bin (2.31-13+deb11u6) over (2.31-13+deb11u3) ...
Preparing to unpack .../libc6_2.31-13+deb11u6_amd64.deb ...
debconf: unable to initialize frontend: Dialog
debconf: (TERM is not set, so the dialog frontend is not usable.)
debconf: falling back to frontend: Readline
debconf: unable to initialize frontend: Dialog
debconf: (TERM is not set, so the dialog frontend is not usable.)
debconf: falling back to frontend: Readline
Unpacking libc6:amd64 (2.31-13+deb11u6) over (2.31-13+deb11u3) ...
Setting up libc6:amd64 (2.31-13+deb11u6) ...
debconf: unable to initialize frontend: Dialog
debconf: (TERM is not set, so the dialog frontend is not usable.)
debconf: falling back to frontend: Readline
(Reading database ... 15719 files and directories currently installed.)
Preparing to unpack .../libncurses6_6.2+20201114-2+deb11u1_amd64.deb ...
Unpacking libncurses6:amd64 (6.2+20201114-2+deb11u1) over (6.2+20201114-2) ...
Preparing to unpack .../libncursesw6_6.2+20201114-2+deb11u1_amd64.deb ...
Unpacking libncursesw6:amd64 (6.2+20201114-2+deb11u1) over (6.2+20201114-2) ...
Preparing to unpack .../libtinfo6_6.2+20201114-2+deb11u1_amd64.deb ...
Unpacking libtinfo6:amd64 (6.2+20201114-2+deb11u1) over (6.2+20201114-2) ...
Setting up libtinfo6:amd64 (6.2+20201114-2+deb11u1) ...
Selecting previously unselected package libpipeline1:amd64.
(Reading database ... 15719 files and directories currently installed.)
Preparing to unpack .../00-libpipeline1_1.5.3-1_amd64.deb ...
Unpacking libpipeline1:amd64 (1.5.3-1) ...
Selecting previously unselected package binfmt-support.
Preparing to unpack .../01-binfmt-support_2.2.1-1+deb11u1_amd64.deb ...
Unpacking binfmt-support (2.2.1-1+deb11u1) ...
Selecting previously unselected package libicu67:amd64.
Preparing to unpack .../02-libicu67_67.1-7_amd64.deb ...
Unpacking libicu67:amd64 (67.1-7) ...
Selecting previously unselected package libxml2:amd64.
Preparing to unpack .../03-libxml2_2.9.10+dfsg-6.7+deb11u4_amd64.deb ...
Unpacking libxml2:amd64 (2.9.10+dfsg-6.7+deb11u4) ...
Selecting previously unselected package libz3-4:amd64.
Preparing to unpack .../04-libz3-4_4.8.10-1_amd64.deb ...
Unpacking libz3-4:amd64 (4.8.10-1) ...
Selecting previously unselected package libllvm13:amd64.
Preparing to unpack .../05-libllvm13_1%3a13.0.1-6~deb11u1_amd64.deb ...
Unpacking libllvm13:amd64 (1:13.0.1-6~deb11u1) ...
Selecting previously unselected package libclang-cpp13.
Preparing to unpack .../06-libclang-cpp13_1%3a13.0.1-6~deb11u1_amd64.deb ...
Unpacking libclang-cpp13 (1:13.0.1-6~deb11u1) ...
Selecting previously unselected package libgc1:amd64.
Preparing to unpack .../07-libgc1_1%3a8.0.4-3_amd64.deb ...
Unpacking libgc1:amd64 (1:8.0.4-3) ...
Selecting previously unselected package libobjc4:amd64.
Preparing to unpack .../08-libobjc4_10.2.1-6_amd64.deb ...
Unpacking libobjc4:amd64 (10.2.1-6) ...
Selecting previously unselected package libobjc-10-dev:amd64.
Preparing to unpack .../09-libobjc-10-dev_10.2.1-6_amd64.deb ...
Unpacking libobjc-10-dev:amd64 (10.2.1-6) ...
Selecting previously unselected package libc6-i386.
Preparing to unpack .../10-libc6-i386_2.31-13+deb11u6_amd64.deb ...
Unpacking libc6-i386 (2.31-13+deb11u6) ...
Selecting previously unselected package lib32gcc-s1.
Preparing to unpack .../11-lib32gcc-s1_10.2.1-6_amd64.deb ...
Unpacking lib32gcc-s1 (10.2.1-6) ...
Selecting previously unselected package lib32stdc++6.
Preparing to unpack .../12-lib32stdc++6_10.2.1-6_amd64.deb ...
Unpacking lib32stdc++6 (10.2.1-6) ...
Selecting previously unselected package libclang-common-13-dev.
Preparing to unpack .../13-libclang-common-13-dev_1%3a13.0.1-6~deb11u1_amd64.deb ...
Unpacking libclang-common-13-dev (1:13.0.1-6~deb11u1) ...
Selecting previously unselected package llvm-13-linker-tools.
Preparing to unpack .../14-llvm-13-linker-tools_1%3a13.0.1-6~deb11u1_amd64.deb ...
Unpacking llvm-13-linker-tools (1:13.0.1-6~deb11u1) ...
Selecting previously unselected package libclang1-13.
Preparing to unpack .../15-libclang1-13_1%3a13.0.1-6~deb11u1_amd64.deb ...
Unpacking libclang1-13 (1:13.0.1-6~deb11u1) ...
Selecting previously unselected package clang-13.
Preparing to unpack .../16-clang-13_1%3a13.0.1-6~deb11u1_amd64.deb ...
Unpacking clang-13 (1:13.0.1-6~deb11u1) ...
Preparing to unpack .../17-git_1%3a2.30.2-1+deb11u2_amd64.deb ...
Unpacking git (1:2.30.2-1+deb11u2) over (1:2.30.2-1) ...
Selecting previously unselected package icu-devtools.
Preparing to unpack .../18-icu-devtools_67.1-7_amd64.deb ...
Unpacking icu-devtools (67.1-7) ...
Selecting previously unselected package libllvm11:amd64.
Preparing to unpack .../19-libllvm11_1%3a11.0.1-2_amd64.deb ...
Unpacking libllvm11:amd64 (1:11.0.1-2) ...
Selecting previously unselected package libclang-cpp11.
Preparing to unpack .../20-libclang-cpp11_1%3a11.0.1-2_amd64.deb ...
Unpacking libclang-cpp11 (1:11.0.1-2) ...
Selecting previously unselected package libffi-dev:amd64.
Preparing to unpack .../21-libffi-dev_3.3-6_amd64.deb ...
Unpacking libffi-dev:amd64 (3.3-6) ...
Selecting previously unselected package libicu-dev:amd64.
Preparing to unpack .../22-libicu-dev_67.1-7_amd64.deb ...
Unpacking libicu-dev:amd64 (67.1-7) ...
Selecting previously unselected package libncurses-dev:amd64.
Preparing to unpack .../23-libncurses-dev_6.2+20201114-2+deb11u1_amd64.deb ...
Unpacking libncurses-dev:amd64 (6.2+20201114-2+deb11u1) ...
Selecting previously unselected package libpfm4:amd64.
Preparing to unpack .../24-libpfm4_4.11.1+git32-gd0b85fb-1_amd64.deb ...
Unpacking libpfm4:amd64 (4.11.1+git32-gd0b85fb-1) ...
Selecting previously unselected package libtinfo-dev:amd64.
Preparing to unpack .../25-libtinfo-dev_6.2+20201114-2+deb11u1_amd64.deb ...
Unpacking libtinfo-dev:amd64 (6.2+20201114-2+deb11u1) ...
Selecting previously unselected package libxml2-dev:amd64.
Preparing to unpack .../26-libxml2-dev_2.9.10+dfsg-6.7+deb11u4_amd64.deb ...
Unpacking libxml2-dev:amd64 (2.9.10+dfsg-6.7+deb11u4) ...
Selecting previously unselected package libyaml-0-2:amd64.
Preparing to unpack .../27-libyaml-0-2_0.2.2-1_amd64.deb ...
Unpacking libyaml-0-2:amd64 (0.2.2-1) ...
Selecting previously unselected package libz3-dev:amd64.
Preparing to unpack .../28-libz3-dev_4.8.10-1_amd64.deb ...
Unpacking libz3-dev:amd64 (4.8.10-1) ...
Selecting previously unselected package llvm-11-runtime.
Preparing to unpack .../29-llvm-11-runtime_1%3a11.0.1-2_amd64.deb ...
Unpacking llvm-11-runtime (1:11.0.1-2) ...
Selecting previously unselected package llvm-runtime.
Preparing to unpack .../30-llvm-runtime_1%3a11.0-51+nmu5_amd64.deb ...
Unpacking llvm-runtime (1:11.0-51+nmu5) ...
Selecting previously unselected package llvm-11.
Preparing to unpack .../31-llvm-11_1%3a11.0.1-2_amd64.deb ...
Unpacking llvm-11 (1:11.0.1-2) ...
Selecting previously unselected package llvm.
Preparing to unpack .../32-llvm_1%3a11.0-51+nmu5_amd64.deb ...
Unpacking llvm (1:11.0-51+nmu5) ...
Selecting previously unselected package python3-pkg-resources.
Preparing to unpack .../33-python3-pkg-resources_52.0.0-4_all.deb ...
Unpacking python3-pkg-resources (52.0.0-4) ...
Selecting previously unselected package python3-pygments.
Preparing to unpack .../34-python3-pygments_2.7.1+dfsg-2.1_all.deb ...
Unpacking python3-pygments (2.7.1+dfsg-2.1) ...
Selecting previously unselected package python3-yaml.
Preparing to unpack .../35-python3-yaml_5.3.1-5_amd64.deb ...
Unpacking python3-yaml (5.3.1-5) ...
Selecting previously unselected package llvm-11-tools.
Preparing to unpack .../36-llvm-11-tools_1%3a11.0.1-2_amd64.deb ...
Unpacking llvm-11-tools (1:11.0.1-2) ...
Selecting previously unselected package llvm-11-dev.
Preparing to unpack .../37-llvm-11-dev_1%3a11.0.1-2_amd64.deb ...
Unpacking llvm-11-dev (1:11.0.1-2) ...
Selecting previously unselected package llvm-13-runtime.
Preparing to unpack .../38-llvm-13-runtime_1%3a13.0.1-6~deb11u1_amd64.deb ...
Unpacking llvm-13-runtime (1:13.0.1-6~deb11u1) ...
Selecting previously unselected package llvm-13.
Preparing to unpack .../39-llvm-13_1%3a13.0.1-6~deb11u1_amd64.deb ...
Unpacking llvm-13 (1:13.0.1-6~deb11u1) ...
Selecting previously unselected package llvm-13-tools.
Preparing to unpack .../40-llvm-13-tools_1%3a13.0.1-6~deb11u1_amd64.deb ...
Unpacking llvm-13-tools (1:13.0.1-6~deb11u1) ...
Selecting previously unselected package llvm-13-dev.
Preparing to unpack .../41-llvm-13-dev_1%3a13.0.1-6~deb11u1_amd64.deb ...
Unpacking llvm-13-dev (1:13.0.1-6~deb11u1) ...
Setting up python3-pkg-resources (52.0.0-4) ...
Setting up libpipeline1:amd64 (1.5.3-1) ...
Setting up libicu67:amd64 (67.1-7) ...
Setting up libyaml-0-2:amd64 (0.2.2-1) ...
Setting up python3-yaml (5.3.1-5) ...
Setting up libffi-dev:amd64 (3.3-6) ...
Setting up python3-pygments (2.7.1+dfsg-2.1) ...
Setting up libz3-4:amd64 (4.8.10-1) ...
Setting up libpfm4:amd64 (4.11.1+git32-gd0b85fb-1) ...
Setting up libllvm11:amd64 (1:11.0.1-2) ...
Setting up libncurses6:amd64 (6.2+20201114-2+deb11u1) ...
Setting up binfmt-support (2.2.1-1+deb11u1) ...
invoke-rc.d: could not determine current runlevel
invoke-rc.d: policy-rc.d denied execution of start.
Setting up icu-devtools (67.1-7) ...
Setting up git (1:2.30.2-1+deb11u2) ...
Setting up libncursesw6:amd64 (6.2+20201114-2+deb11u1) ...
Setting up libgc1:amd64 (1:8.0.4-3) ...
Setting up libc6-i386 (2.31-13+deb11u6) ...
Setting up libc-dev-bin (2.31-13+deb11u6) ...
Setting up libxml2:amd64 (2.9.10+dfsg-6.7+deb11u4) ...
Setting up libz3-dev:amd64 (4.8.10-1) ...
Setting up llvm-11-runtime (1:11.0.1-2) ...
Setting up llvm-11-tools (1:11.0.1-2) ...
Setting up llvm-11 (1:11.0.1-2) ...
Setting up libobjc4:amd64 (10.2.1-6) ...
Setting up libclang-cpp11 (1:11.0.1-2) ...
Setting up lib32gcc-s1 (10.2.1-6) ...
Setting up lib32stdc++6 (10.2.1-6) ...
Setting up llvm-runtime (1:11.0-51+nmu5) ...
Setting up llvm (1:11.0-51+nmu5) ...
Setting up libllvm13:amd64 (1:13.0.1-6~deb11u1) ...
Setting up libobjc-10-dev:amd64 (10.2.1-6) ...
Setting up llvm-13-runtime (1:13.0.1-6~deb11u1) ...
Setting up libc6-dev:amd64 (2.31-13+deb11u6) ...
Setting up libicu-dev:amd64 (67.1-7) ...
Setting up libclang-common-13-dev (1:13.0.1-6~deb11u1) ...
Setting up libncurses-dev:amd64 (6.2+20201114-2+deb11u1) ...
Setting up llvm-11-dev (1:11.0.1-2) ...
Setting up llvm-13-linker-tools (1:13.0.1-6~deb11u1) ...
Setting up llvm-13-tools (1:13.0.1-6~deb11u1) ...
Setting up libclang1-13 (1:13.0.1-6~deb11u1) ...
Setting up libxml2-dev:amd64 (2.9.10+dfsg-6.7+deb11u4) ...
Setting up libclang-cpp13 (1:13.0.1-6~deb11u1) ...
Setting up libtinfo-dev:amd64 (6.2+20201114-2+deb11u1) ...
Setting up clang-13 (1:13.0.1-6~deb11u1) ...
Setting up llvm-13 (1:13.0.1-6~deb11u1) ...
Setting up llvm-13-dev (1:13.0.1-6~deb11u1) ...
Processing triggers for libc-bin (2.31-13+deb11u3) ...
ln: failed to create symbolic link './clang': File exists
make: *** [Makefile:29: release] Error 1
```

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -9,7 +9,7 @@ on:
 jobs:
 
   build:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
     steps:
     - uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9
 
@@ -35,7 +35,7 @@ jobs:
     - name: Install LLVM and Clang
       uses: KyleMayes/install-llvm-action@f8f2154d96f018dcb600739c4978bfc35f435422
       with:
-        ubuntu-version: "20.04"
+        ubuntu-version: "22.04"
         version: "12.0.0"
         directory: $HOME/.clang
         cached: ${{ steps.cache-llvm.outputs.cache-hit }}

--- a/Makefile
+++ b/Makefile
@@ -30,8 +30,8 @@ release:
 		--rm \
 		--workdir /pwru \
 		--volume `pwd`:/pwru docker.io/library/golang:1.20.5 \
-		sh -c "apt update && apt install -y make git clang-13 llvm && \
-			ln -s $(which clang-13) /usr/bin/clang && \
+		sh -c "apt update && apt install -y make git clang-13 llvm curl unzip gcc flex bison && \
+			ln -s /usr/bin/clang-13 /usr/bin/clang && \
 			git config --global --add safe.directory /pwru && \
 			make libpcap.a && \
 			make local-release VERSION=${VERSION}"

--- a/Makefile
+++ b/Makefile
@@ -16,14 +16,12 @@ $(TARGET):
 		-X 'github.com/cilium/pwru/internal/pwru.Version=${VERSION}'"
 
 libpcap.a:
-	apt update && apt-get install -y curl unzip gcc flex bison make && \
-        curl https://github.com/the-tcpdump-group/libpcap/archive/refs/tags/libpcap-1.10.4.zip -OL && \
-        unzip -o libpcap-1.10.4.zip && \
-        cd libpcap-libpcap-1.10.4/ && \
-	./configure --enable-dbus=no && \
-	make && \
-	make install
-
+	curl https://github.com/the-tcpdump-group/libpcap/archive/refs/tags/libpcap-1.10.4.zip -OL
+	unzip -o libpcap-1.10.4.zip
+	cd libpcap-libpcap-1.10.4/ && \
+		./configure --disable-shared --disable-usb --disable-netmap --disable-bluetooth --disable-dbus --without-libnl && \
+		make && \
+		make install
 
 release:
 	docker run \

